bundle/config/mutator/apply_presets.go

```diff
@@ -221,6 +222,19 @@ func (m *applyPresets) Apply(ctx context.Context, b *bundle.Bundle) diag.Diagnos
 		dashboard.DisplayName = prefix + dashboard.DisplayName
 	}
 
+	if config.IsExplicitlyEnabled((b.Config.Presets.InPlaceDeployment)) {
```

Comment:
Why is `InPlaceDeployment` a present and a configurable in the bundle YAML? Do we want to make it opt-in for users? 

If not it might make sense to use an environment variable as a flag to control this.